README.md

```diff
@@ -85,7 +85,11 @@ This is built on the Arduino platform for easy and plentiful I/O. Arduino provid
 
 The rocket engine has 8 sensors:
 - 1 Load Cell for measuring thrust
-- 1 Thermocouple placed before the injector plate
+- 4 Thermocouples
+    - 1 Inside the Combustion Chamber
```

Comment:
1. At the Nozzle exit
1. On the Kerosene Inlet and Injector Plate
1. On the N2O inlet and injector Plate
1. On the combustion chamber at the throat of the nozzle (the constriction)
![image](https://github.com/user-attachments/assets/65f1e728-bc12-4ae2-830a-f3080d1fb4bf)